opensanctions/crawlers/md_interdictie.py

```diff
@@ -115,15 +135,17 @@ def parse_date(text):
     return h.parse_date(text, ["%d/%m/%Y", "%d.%m.%Y", "%A, %d %B, %Y"])
 
 
-def parse_sanction_num(context, text) -> str | None:
+def parse_sanction_decision(context, text):
     match = REGEX_SANCTION_NUMBER.match(text)
     if match:
-        return match.group(1)
+        return match.group(1), parse_date(match.group(2))
     else:
-        context.log.warn(f'Failed to parse saction number from "{ text }"')
+        context.log.warn(f'Failed to parse saction number and date from "{ text }"')
+        return None, None
 
 
 def parse_address(context, text):
+    """Currently just the plain string as full address"""
```

Comment:
If we do this, it'll probably live upstream (i.e. use libpostal). I'd just get rid of the function.